openghg/standardise/_standardise.py

```diff
@@ -875,3 +875,99 @@ def standardise_from_binary_data(
             binary_data=binary_data, metadata=metadata, file_metadata=file_metadata, **kwargs
         )
     return result
+
+
+def standardise_flux_timeseries(
+    filepath: Union[str, Path],
+    species: str,
+    source: str,
+    domain: Optional[str] = None,
+    region: str = "UK",
```

Comment:
Picky comment but I think the keywords should be rearranged here to emphasise importance:
```
    region: str = "UK",
    source_format: str = "crf",
    domain: Optional[str] = None,
```

For now we can leave the defaults of "crf" and "UK" but if we add any other flux_timeseries parser functions we should also remove these defaults.

I think if we have `domain` above `region` this will get in preference and in circumstances when it doesn't make the most sense.